src/read.c

```diff
@@ -2756,6 +2762,23 @@ static avifResult avifParseItemPropertyAssociation(avifMeta * meta, const uint8_
             // Copy property to item
             const avifProperty * srcProp = &meta->properties.prop[propertyIndex];
 
+            // ISO/IEC 23000-22 Section 6.5.1:
```

Comment:
6.5.1 is not in ISO/IEC 23000-22:2019 (the [current published version](https://www.iso.org/standard/74417.html)).

This requirement is found in ISO/IEC 23000-22:2019/Amd. 2:2021 Section 7.3.9. Note the amendment 2.

Include the version/edition number of ISO/IEC 23008-12 that you are quoting from.

Note: In line 2818 you have Section 7.3.9, which is correct:

```
                        // MIAF: Section 7.3.9: "All transformative properties associated with coded and derived
                        //                      images shall be marked as essential"
```

Reply:
This depends on whether section/clause numbers may change in a new version of the standard. Do you know if these standards try to keep their section/clause numbers fixed?